src/data/navigation/sections/rest.js

```diff
@@ -8,13 +8,31 @@ module.exports = [
         path: "/rest/quick-reference/",
     },
     {
-        title: "Generate a local API Reference",
-        path: "/rest/quick-reference/generate-local.md",
+        title: "Authentication",
+        path: "/rest/authentication/",
+        pages: [
+            {
+                title: "User authentication",
+                path: "/rest/authentication/user.md",
+            },
+            {
+                title: "Server-to-server authentication",
+                path: "/rest/authentication/server-to-server.md",
+            }
+        ],
+    },
```

Comment:
Should we add a link in the auth folder to the PaaS auth?
e.g. [PaaS REST authentication](https://developer.adobe.com/commerce/webapi/get-started/authentication/)

Reply:
I added a link to the PaaS auth overview topic in the SaaS auth overview topic instead.